pkg/storage/util_test.go

```diff
@@ -152,7 +152,13 @@ func (m *mockChunkStore) GetChunkRefs(ctx context.Context, userID string, from,
 		}
 		refs = append(refs, r)
 	}
-	f, err := chunk.NewChunkFetcher(cache.Config{}, false, m.client)
+
+	cache, err := cache.New(cache.Config{Prefix: "chunks"})
```

Comment:
the prefix was previously set in `chunk.NewChunkFetcher`. I like the explicit nature of this more 👍